addons/account/data/digest_data.xml

```diff
@@ -5,4 +5,19 @@
             <field name="kpi_account_total_revenue">True</field>
         </record>
     </data>
+
+    <data>
+        <record id="digest_tip_account_0" model="digest.tip">
+            <field name="name">Tip: No need to print, put in an envelop and post your invoices</field>
+            <field name="sequence">700</field>
+            <field name="group_id" ref="account.group_account_invoice" />
+            <field name="tip_description" type="html">
+                <div>
```

Comment:
Linting note: for html content, you can de-indent until the beginning of the file to save space, like
```
        <field name="tip_description" type="html">
<div>
    <p>BLahblahb</p>
    <p>Bloublou</p>
</div>
        </field>
```